helpers/send_report_to_admins.js

```diff
@@ -0,0 +1,64 @@
+module.exports = function(type) {
+  var nodemailer = require('nodemailer'),
+  smtpTransport = require('nodemailer-smtp-transport'),
+  markdown = require('nodemailer-markdown').markdown,
+  Admin = require('../models/admin'),
+  BugReport = require('../models/bug_report'),
+  CrashReport = require('../models/crash_report');
+
+  return new Promise((function(resolve, reject) {
+    Admin.find(function(error, admins) {
+      if(error)
+        return reject(error);
+
+      var emails = admins.map(function(admin) {
+        return admin.email;
+      });
+      var classObject = 'type' == 'Bug' ? BugReport : CrashReport;
+      var limit = {};
```

Comment:
This is where it should throttle, e.g. `{"created_at": {"$gte": lastSent}}`. If there's no last, then just send the current object.